src/Maestro/maestro-angular/src/maestro-client/maestro.ts

```diff
@@ -1123,6 +1130,46 @@ export class ChannelsApiService implements IChannelsApi {
         );
 
     }
+
+    public getFlowGraphAsync(
+        {
+            id,
+        }: {
+            id: number,
+        }
```

Comment:
We could make local interface for `{id:number}`. That will make a few of these lines less awkward. :-)

Reply:
This is another place where I copied getBuildGraphAsync and followed that formatting. I'm also not quite clear what you mean here.